rust.exp

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/expect -f
```

Comment:
This file is partially copied from https://github.com/apache/incubator-teaclave-trustzone-sdk/blob/master/ci/qemu-check.exp so please add this link to the commit description.